project-management/Sprint_Reports/Sprint_1.md

```diff
@@ -0,0 +1,82 @@
+# Overview of CSP 
+
+## Functional Product:
+
+Adaptive Wireframes 
+https://c6q95y.axshare.com
+
+Experience Screens 
+https://xsswhw.axshare.com 
+
+UX Deliverables 
+https://blm1j3.axshare.com
+
+Tom's Review
+https://2huje4.axshare.com/
+
+## Board Issues
+
+### Issues Closed
+* #: Waiting on feedback from community on core MVP - Components outlined below
+*	#: 
+
+### Issues Progressed
+Progress was made on these tasks but they were not closed during the sprint.
+*	#: Cross Platform 
+*	#: Caregiver Platform Access - User Type / UX Deliverables Persona / User  Journey Explored 
+*	#: Interface Guidance 
+* #: Navigation 
+*	#: Data Entry Area (Autistic User Add a Experience / View ) 
+*	#: Delete Published Experience
+*	#: UI / Colour Themes / Font as UX Deliverables Link - UI Kit 
+*	#: Platform Requirements - Established some further learning required around GDPR / what support will be required from OH
+*	#: Research Notification - Considered in User Journey Initially
+*	#: Enable user to make a choice about allowing researchers to make further contact - Considered in User Journey screen to be wireframes in next sprint following community review and feedback of this demo 
+
+## MVP  (Home Page Components)
+*	#: URL Defined 
+*	#: Header - Taking on own identity or that of similar to OH to create consistent experience for the user (demo screens)
+*	#: Hero - Aims To be Refined and aligned to project 
+*	#: Constant Navigation - Initial navigation / titles added linked to wireframes and user journey 
+*	#: Home page / Title / Aims / Core Functions  
+*	#: Simple Clean Design - Flowing Content 
+*	#: Accessibility Theme 
+*	#: Sub Footer Safe & Secure - Tool Tips 
+*	#: Footer - Copyright 
+
+User Journey Linked to Personas within UX Deliverables Presentation 
+https://blm1j3.axshare.com
+
+## UX Deliverables Content 
+
+* Persona Wireframes
+* User Story Mapping
+* Information Architecture
+* Designing For Autism 
+* Colour Scheme Refs
+* Support Material 
+* Design Thinking 
```

Comment:
I don't really know what I'm reading here - is this work that was done in sprint 1 but not described in issues? 

I think the best way to incorporate this work is to make sure that there _are_ issues for each of the tasks and then put them all under the `### Issues progressed` section as described in the SPRINTREPORT file.

Reply:
These were all covered in the issues. Apologies for the duplication I think Chris was trying to give more detail than necessary about the UX work he did.